src/Raven.Studio/typescript/components/pages/resources/databases/store/databasesViewSelectors.ts

```diff
@@ -74,8 +74,6 @@ const selectFilterByStateOptions = (store: RootState): InputItem<DatabaseFilterB
         { value: "Offline", label: "Offline", count: offline },
         { value: "Error", label: "Errored", count: error },
         { value: "Disabled", label: "Disabled", count: disabled },
-        { value: "Sharded", label: "Sharded", count: sharded, verticalSeparatorLine: true },
```

Comment:
do we need sharded / nonSharded variables now?

Reply:
I will remove sharded/nonsharded from filter logic

Reply:
done